tests/Makefile

```diff
@@ -1,14 +1,2 @@
-build-trainer-docker:
-	docker build -t tracee-trainer:latest .
-
-build-tracee-docker-core:
-	cd .. && make -f builder/Makefile.tracee-container build-alpine-tracee-core
-
-build-tracee-docker-nocore:
-	cd .. && make -f builder/Makefile.tracee-container build-alpine-tracee-nocore
-
-build-tracee-docker-core-btfhub:
-	cd .. && make -f builder/Makefile.tracee-container build-alpine-tracee-core-btfhub
-
-run-tests: build-trainer-docker build-tracee-docker-core build-tracee-docker-nocore build-tracee-docker-core-btfhub
-	go test -v -run=TestTraceeSignatures
\ No newline at end of file
+docker-build-tracee-trainer:
+	docker build -t tracee-trainer:latest tests
```

Comment:
Is this correct? 
```vagrant@ubuntu-impish:~/repos/tracee/tests$ make docker-build-tracee-trainer 
docker build -t tracee-trainer:latest tests
unable to prepare context: path "tests" not found
make: *** [Makefile:2: docker-build-tracee-trainer] Error 1
```

Maybe you meant: `docker build -t tracee-trainer:latest .`?

Reply:
Sorry ignore that - I understood your motivation now and where it would be run from.

Reply:
It's a temporary change to run it from the root directory. Relative paths are tricky. We are going to revisit makefile targets with @rafaeldtinoco to make it consistent with other build scripts